src/Test/AbstractFixerTestCase.php

```diff
@@ -161,6 +165,12 @@ protected function doTest($expected, $input = null, \SplFileInfo $file = null, F
         Tokens::clearCache();
         $tokens = Tokens::fromCode($expected);
 
+        if (!$fixer->isCandidate($tokens)) {
+            $this->assertFalse($tokens->isChanged(), 'Fixer should not touch Tokens on candidate check.');
+
+            return;
```

Comment:
no need for return after assertion - if it fail then it throws internal PHPUnit exception to break test execution